presto-hive/pom.xml

```diff
@@ -328,6 +334,7 @@
                         <configuration>
                             <excludes>
                                 <exclude>**/TestHiveGlueMetastore.java</exclude>
+                                <exclude>**/TestGlueJMXStatistics.java</exclude>
```

Comment:
Looks like there's a `test-hive-glue` profile that got dropped when we moved from travis to github workflows?

Reply:
These were never actually used